applications/plugins/picopass/picopass_worker.c

```diff
@@ -291,51 +240,35 @@ static ReturnCode picopass_auth_dict(
 ReturnCode picopass_auth(PicopassBlock* AA1, PicopassPacs* pacs) {
     ReturnCode err;
 
-    FURI_LOG_I(TAG, "Trying standard legacy key");
-    err = picopass_auth_standard(
-        AA1[PICOPASS_CSN_BLOCK_INDEX].data, AA1[PICOPASS_KD_BLOCK_INDEX].data);
-    if(err == ERR_NONE) {
-        memcpy(pacs->key, picopass_iclass_key, PICOPASS_BLOCK_LEN);
-        return ERR_NONE;
-    }
-
-    FURI_LOG_I(TAG, "Trying factory default key");
-    err = picopass_auth_factory(
-        AA1[PICOPASS_CSN_BLOCK_INDEX].data, AA1[PICOPASS_KD_BLOCK_INDEX].data);
-    if(err == ERR_NONE) {
-        memcpy(pacs->key, picopass_factory_debit_key, PICOPASS_BLOCK_LEN);
-        return ERR_NONE;
-    }
-
-    FURI_LOG_I(TAG, "Starting user dictionary attack [Elite KDF]");
+    FURI_LOG_I(TAG, "Starting system dictionary attack [Standard KDF]");
```

Comment:
The diff doesn't make it clear, but I was moving the last block up since the standard and factory keys are system dictionary/standard KDF